src/status_im/ui/screens/chat/toolbar_content.cljs

```diff
@@ -53,3 +53,20 @@
 
 (defn toolbar-content-view []
   [toolbar-content-view-inner @(re-frame/subscribe [:chats/current-chat])])
+
+(defn toolbar-pin-content-view []
```

Comment:
Use `defview` and `letsubs` instead of raw `re-frame/subscribe`

Reply:
honestly i don't think we have a strict rule ,  we need to discuss this, because there are also new `[status-im.utils.handlers :refer [>evt <sub]]`, i guess defview is outdated, because it was using before subscriptions is cashed, so probably we should move to `>evt <sub` the only thing i like in defview is map with component lifecycle handlers